docs/Schema-changes.md

```diff
@@ -0,0 +1,176 @@
+
+#### Credential title
+The credential title is the main attribute you want users to see when viewing a credential. An example of this would be the business number of a business number credential. See figure 1 for an example.
+
+![](./assets/schema_label_cred_title.png)  
+*Figure 1: Schema label and credential title*
+  
+#### Highlighted attributes
+Highlighted attributes are secondary credential attributes that you believe would be important to a user viewing a specific credential. An example of this would be a Cannabis marketing license credential. A user would likely be interested in knowing the effective and expiry date of the credential, but probably would not care much about the issue date since it does not have anything to do with the validity of the credential. Aries VCR allows issuers to include only relevant credential attributes in their credential displays. See figure 2 for an example.  
+
+![](./assets/highlighted_attributes.png)  
+*Figure 2: Highlighted attributes*
+
+#### Schema labels
+Aries VCR has added the functionality to specify the name of a credential schema in a multilingual format. These schema names appear as the title of the credential and let users know what type of credential they are viewing. This schema label is optional and is only needed if you plan on supporting multi language use. If no schema label is defined, a good fallback is to use the credential type description displayed in the same way as a schema label. The new version of Orgbook takes this approach. See figure 1 for example of a schema label in Orgbook.
+  
+#### Aries VCR defaults
+##### Credential title:
+If the credential title is not specified in a credential schema, or there is no claim label to match the credential title, then applications such as Orgbook may instead use the registration number of the entity holding the credential. See figure 3 for an example  
+![](./assets/default_cred_title.png)  
+*Figure 3: default credential title and highlighted attributes*
+  
+##### Highlighted attributes:
+If no highlighted attributes are specified, or there are no claim labels to match the highlighted attributes, then applications like Orgbook will not display any highlighted attributes. See figure 3.1 to see how a credential with no highlighted attributes or credential title is displayed.  
+![](./assets/default_highlighted.png)  
+*Figure 3.1: Default credential title and highlighted attributes*
+
+#### Recommended changes to schema
+To incorporate these optional changes, Aries VCR recommends reviewing the new schema attributes and how to add them to your credential schema in the section below. The following example JSON snippet is from the `issuer_registration.credential_types[item]` section of the registration.registries.ca schema
+
+~~~
```

Comment:
This looks good.  Probably worth pointing out where this format of message is used (is it the format provided to the new issuer agency?) and linking to the demo issuer/controller (https://github.com/bcgov/aries-vcr-issuer-controller/tree/main/issuer_controller/config#changes-for-schema-localization), which has applied a corresponding set of changes, and represents the changes that the current set of issuers will need to make.